### DISCUSSION TITLE
When is version 6.6.0 going to be released?

### DISCUSSION THREAD

**Discussion by JetoPistola** [pyinstaller/pyinstaller#8403] [Features & Development]
Hello Pyinstaller team! 🐍 

We need the new version, for the following reason: 

<img width="930" alt="Screenshot 2024-04-12 at 12 59 59" src="https://github.com/pyinstaller/pyinstaller/assets/9656369/b5c919a6-0da3-4c5a-8caf-241503c0a093">

When is the expected release date? 📅 

If not... we'll need a temporary solution for the following days/weeks/months.

Thanks in advance,

A fellow ninja 🥷  python 🐍 developer  💻 👨‍💻

**Comment by bwoodsend** [comment]
2<sup>nd</sup> Saturday of each month is the reminder I arbitrarily stuck in my phone a year or so ago. So tomorrow.
